modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/getProjectRoles.js

```diff
@@ -36,6 +36,10 @@ export function getRolesFiltered(items, project) {
 						role.name !== ROLE_TYPES.partnerMember.key;
```

Comment:
Hey Pedro, can you add the new roles to this condition?

Reply:
Done.